src/main/java/uk/gov/ons/ctp/response/casesvc/domain/model/CaseAction.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.ons.ctp.response.casesvc.domain.model;
+
+import java.util.UUID;
+import javax.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+import uk.gov.ons.ctp.response.casesvc.representation.CaseGroupStatus;
+
+@Entity
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
I would of expected this to need the @Table annotation to tell it what schema to look in?

```
@Table(name = "case_action", schema = "case")
```

Reply:
Not sure - isn't @Table is the transitional actual table and as it is a view not sure what should it be?